forms-flow-components/src/components/CustomComponents/ConfirmModal.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import Modal from "react-bootstrap/Modal";
+import {CustomButton} from "./Button";
+import { CloseIcon } from "../SvgIcons/index";
+
+interface ConfirmModalProps {
+  show: boolean;
+  onCancel: () => void;
+  title: string;
+  message: string;
+  messageSecondary?: string;
+  onConfirm: () => void;
+  confirmText: string;
```

Comment:
Please change the text from 'confirm' to 'primary' and 'cancel' to 'secondary' in all cases in this file